From 13e8d7b5486d9910fb5ca6abc3cc85162deab675 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Fri, 18 Sep 2015 23:53:03 +0800
Subject: [PATCH 0528/5242] MLK-11566-3 input: keyboard: add i.mx6ul support
 for snvs power key

commit  b7caa9e2cabf29774541df17085383df89ea613f from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6UL has SNVS power key, add support for it.

Signed-off-by: Anson Huang <b20788@freescale.com>
(cherry picked from commit 095c876a12e27220cb91cb9dc376a20b6bf259e4)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/input/keyboard/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/keyboard/Kconfig b/drivers/input/keyboard/Kconfig
index 6bd97ff..df117c8 100644
--- a/drivers/input/keyboard/Kconfig
+++ b/drivers/input/keyboard/Kconfig
@@ -420,7 +420,7 @@ config KEYBOARD_MPR121
 
 config KEYBOARD_SNVS_PWRKEY
 	tristate "IMX SNVS Power Key Driver"
-	depends on SOC_IMX6SX
+	depends on (SOC_IMX6SX || SOC_IMX6UL || SOC_IMX7)
 	depends on OF
 	help
 	  This is the snvs powerkey driver for the Freescale i.MX application
-- 
1.7.9.5

